tracer/src/Datadog.Trace/AppSec/Security.cs

```diff
@@ -316,6 +302,19 @@ private void Report(ITransport transport, Span span, IResult result, bool blocke
             AddHeaderTags(span, headers, RequestHeaders, SpanContextPropagator.HttpRequestHeadersTagPrefix);
         }
 
+        private void SetSamplingPriority(Span span)
+        {
+            if (!_settings.KeepTraces)
```

Comment:
nit -  I would have put that in an else if below

Reply:
I joined the two ifs with an else, as I believe KeepTraces should override other settings, but it makes sense to me to keep it first.